pkg/pfcpsim/pfcpsim.go

```diff
@@ -209,6 +210,22 @@ func (c *PFCPClient) SendSessionEstablishmentRequest(pdrs []*ieLib.IE, fars []*i
 	return c.sendMsg(estReq)
 }
 
+func (c *PFCPClient) SendSessionModificationRequest(PeerSEID uint64, pdrs []*ieLib.IE, qers []*ieLib.IE ,fars []*ieLib.IE) error {
```

Comment:
As agreed offline, the PR should only contain `SendSessionModificationRequest`. Please split this PR.

Reply:
Done